docs/soroban-cli-full-docs.md

```diff
@@ -65,6 +67,8 @@ This document contains the help content for the `soroban` command-line program.
 * [`soroban network add`↴](#soroban-network-add)
 * [`soroban network rm`↴](#soroban-network-rm)
 * [`soroban network ls`↴](#soroban-network-ls)
+* [`soroban network start`↴](#soroban-network-start)
+* [`soroban network stop`↴](#soroban-network-stop)
```

Comment:
Something that might be confusing with bundling this under `network` is that folks configure a network, and then we also use network to mean this thing which isn't the same thing. In fact we can configure networks called the same name that aren't the same.

Looking at some other tools in other ecosystems, ganache calls these things "instances" instead of "network". Maybe it would make sense to use a similar term, a term that doesn't imply this thing is a network, but a local instance.

Or maybe there's another option. Or maybe not confusing for others...

Reply:
@leighmcculloch Hm, interesting point. Just to make sure I'm understanding correctly - you're talking about the distinction between:
- this PR, which is starting up nodes instances for different networks vs
- the other things that the `network` subcommand is doing is configuring connections to different networks

is that right?

I wonder if it would be worth putting this functionality under `soroban network instance`, so that is is more clear that this is an instance of node that is connecting to a specific network? So it would be `soroban network instance start testnet`. 

What do you think @chadoh ?

Reply:
@leighmcculloch Could you suggest what the subcommand could be? Thanks

Reply:
I was thinking `soroban instance list|start|stop`.

But `soroban network instance ...` could work too, especially if launching an instance auto configures a network for that instance.

Reply:
Based on the conversation in the meeting last week, it seems like folks were good with moving forward with `soroban network start | stop`. I'll plan to go ahead with that, and can always make an update in the future.

Reply:
Added a link to this comment conversation in the follow up issue: https://github.com/orgs/stellar/projects/50?pane=issue&itemId=54158722